src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -90,6 +89,11 @@ public void setPower(double power) {
 
     @Override
     public Distance getCurrentValue() {
+        Distance currentHeight = Inches.of(0);
+        if (contract.isElevatorReady()){
+            double rotations = this.masterMotor.getPosition().in(Degrees) / 360;
```

Comment:
⭐ ⭐ There's an easier way of doing this with units directly
```suggestion
            double rotations = this.masterMotor.getPosition().in(Rotations);